### ISSUE TITLE
showToast close button missing aria-label="Close"

### ISSUE THREAD

**Issue by umeWaheed** [siemens/ix#1940]
### Prerequisites

- [x] I have read the [Contributing Guidelines](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md).
- [x] I have not leaked any internal/restricted information like screenshots, videos, code snippets, links etc.

### What happened?

I'm rendering toasts using the showToast method and it used to have `aria-label="close"` for the close button, which we were using for testing.

After upgrading to v3 the aria-label is set as `aria-label="Unknown"`

![Image](https://github.com/user-attachments/assets/266c9029-3f64-4677-91a3-c8176b0baa54)

### What type of frontend framework are you seeing the problem on?

React

### Which version of iX do you use?

v3

### Code to produce this issue.

```shell
https://ix.siemens.io/docs/components/toast/code
```
